py_swf/config_definitions.py

```diff
@@ -7,11 +7,27 @@
 
 WorkflowClientConfig = namedtuple(
     'WorkflowClientConfig',
-    'domain workflow_name workflow_version task_list execution_start_to_close_timeout task_start_to_close_timeout',
+    'domain task_list execution_start_to_close_timeout task_start_to_close_timeout',
 )
 """An immutable object that stores common SWF values. Used by instances of :class:`~py_swf.clients.workflow.WorkflowClient`.
 """
 
+StartWorkflowResult = namedtuple(
```

Comment:
These aren't config definitions. I'd probably define these in the same file where they're used.